docs/input.rst

```diff
@@ -101,6 +101,19 @@ using :meth:`SampleIterator.get_number()`, :meth:`SampleIterator.get_bool()` and
 
 See more information in :ref:`Accessing the data`.
 
+Accessing the SampleInfo
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can access the meta-data of a sample, the SampleInfo, as follows:
```

Comment:
You can access *a field of* the meta-data of a sample,
(Or change the example to obtain the entire object instead of a field)